rabbitmq_workers/worker/miners/scielo.py

```diff
@@ -173,24 +191,23 @@ def gather(self, url):
                 self.results = None
         pass
 
+# Static miner from this file
+miner = ScieloEngine(ScieloLocations)
+
 def GetArticle(url):
-    miner = ScieloEngine(ScieloLocations)
     miner.gather(url)
+
+    # Check for valid data
+    if miner.results is None:
+        return None
     
-    # Mould results to match schema
-    references = miner.results.pop('references')
-    keywords = miner.results.pop("keywords")
-    orgs = miner.results.pop("organization_affiliated")
-    license = miner.results.pop("license")
-    extra = miner.results.pop('extra_link')
-
-    meta_info = {
-        "references": references,
-        "keywords": keywords,
-        "organization_affiliated": orgs,
-        "license": license,
-        "pdf_link": extra
-    }
+    # Mould results to match schema - move extra data to the meta_info column
+    meta_info = {}
+    keys = ['references', 'organization_affiliated', 'keywords', 'license', 'extra_link']
+    for key in keys:
+        if miner.results.get(key) is not None: # key exists and is non-None
+            meta_info = miner.results.pop(key)
+
     miner.results['meta_info'] = json.dumps(meta_info)
```

Comment:
should we add `json.dumps(meta_info, ensure_ascii=False)` for non-English articles?